cinch/roles/jenkins_slave_container_wrapup/templates/jswarm.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash -xe
-. /etc/sysconfig/jenkins_swarm.templated
+. /var/lib/jenkins/jenkins_swarm.templated
```

Comment:
/var/lib/jenkins is our $HOME for the jenkins user. The dockerize command has to execute as the jenkins user for use in OpenShift, so we need to have the location of the jenkins_swarm.templated file be writable by that user.

I defaulted to putting it in that user's home directory, but it's definitely not the only viable choice. We could pre-create an empty file at /etc/sysconfig/jenkins_swarm.templated that is owned by jenkins in the container cleanup role and leave the file there, or we could change permissions on some other folder for a container as well. Doing so would save us problems if we ever change the default value of {{ jenkins_user_home }}.